released/packages/coq-num-analysis/coq-num-analysis.1.0.0/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
+synopsis: "Numerical Analysis in Coq"
+name: "coq-num-analysis"
+version: "1.0.0"
```

Comment:
The package name and version is already given by the file path (`coq-num-analysis.1.0.0/opam`), so avoid them inside the file.
```suggestion
```